src/test/scala/info/bethard/timenorm/formal/ReadersTest.scala

```diff
@@ -174,7 +184,10 @@ class ReadersTest extends FunSuite with TypesSuite {
         </annotations>
       </data>.convert
     implicit val data = Data(xml, "last few months")
-    val temporals = data.entities.map(AnaforaReader.temporal)
+    val start = LocalDateTime.now().truncatedTo(DAYS)
+    val dct = SimpleInterval(start, start.plusDays(1))
+    val aReader = new AnaforaReader(dct)
+    val temporals = data.entities.map(aReader.temporal)
     temporals match {
       case Seq(last: Last, _: Number, _: RepeatingInterval) => assert(!last.isDefined)
```

Comment:
Yes, with your revisions to add the DCT to AnaforaReader, this should be updated to say `assert(last.isDefined)`. Even better, rather than using `.now()`, set `start` to `LocalDate.of(1998, 3, 6, 0, 0)` and check that `last === SimpleInterval(...)`. You'll want to do the same thing for the `NYT19980206.0460` test too, which is likely to fail for the same reason.